styles/main/_logs_diagnostics.scss

```diff
@@ -12,7 +12,8 @@
 		-moz-user-select: text;
 		-ms-user-select: text;
 		user-select: text;
-		width: fit-content;
+		overflow-x: scroll;
+		overflow-y: hidden;
```

Comment:
Why don't you want the user to scroll down to older log messages?

Moreover, `<pre>` is used in a block-formatting context and is a block-level element, i.e. its intrinsic height always matches the content (unless we have changed that somewhere, I need to check that) This means the CSS property `overflow-y` won't have any effect unless `height` is specified, too. (Again I need to check first, if we do so somewhere.)